image/image.go

```diff
@@ -26,6 +26,9 @@ import (
 // validates the manifest pointed to by the given ref
 // or returns an error if the validation failed.
 func ValidateLayout(src, ref string) error {
+	if _, err := os.Stat(src); err != nil {
```

Comment:
@jonboulle excuse me, I don't fully understand what you mean. This check should be done because that `no file exist case` should be checked and returned as early as possible, to avoid the future redundant program being executed.